spec/dummy_app/app/models/callback_modifier.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class CallbackModifier < ActiveRecord::Base
+  # These guys need to look just a _little_ more like a real STI class in order for the tests to
+  # call modifier.versions().
+  attr_accessor :type
```

Comment:
This seems highly unusual. Is this how your app works, Sean? I've never heard of this technique and I'm not surprised that PT was confused by it. My gut reaction is that the amount of complexity I'd be willing to introduce into PT to support this use-case is minimal. I certainly am not interested in patching rails to support this.

Reply:
This is in relation to the workaround that triggers pre-1108 behaviour.

Perhaps we can have a flag so people can choose base_class or real class.  I'm not totally onboard with us always storing both.